lib/wanda_web/views/health_json.ex

```diff
@@ -0,0 +1,3 @@
+defmodule WandaWeb.HealthJSON do
+  def health(%{health: health}), do: health
```

Comment:
Wit this change, don't you need to change `health_controller.ex` file, to change `health.json` simply by `:health` ?
Something like `render(:health, health: %{database: db_status})`

Reply:
Oh god yes!